db/changes.go

```diff
@@ -774,9 +774,13 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			}
 
 			if options.Revocations && db.user != nil {
-				channelsToRevoke := db.user.RevokedChannels(options.Since.SafeSequence())
-				for channel, triggeredBy := range channelsToRevoke {
-					feed := db.buildRevokedFeed(db.changeCache.getChannelCache().getSingleChannelCache(channel), options, triggeredBy, to)
+				revocationCheckSeq := options.Since.Seq
```

Comment:
Keep SafeSequence(), or see whether RevokedChannels should do that work.

Might be useful to throw a few cases of sequences with lowSeq in the tests as since values, to make sure it's not missed elsewhere.

Reply:
Switched back to SafeSeq and added a test case in the existing tests.